src/Viewer/components/MenuBar/MenuBar.tsx

```diff
@@ -17,10 +25,28 @@ import NavigationBar from "./NavigationBar";
 import "./MenuBar.scss";
 
 
+dayjs.extend(timezone);
```

Comment:
I believe the extension registrations are already done in App.tsx. Do we have to add those in MenuBar.jsx?